src/services/base.service.js

```diff
@@ -114,14 +119,30 @@ const baseSearchById = async (args, resourceType) => {
  * @param {*} req The Express request object. This is used by the query builder.
  * @param {*} resourceType The resource type we are searching on.
  * @param {*} paramDefs Optional parameter definitions for the specific resource types. Specific
- *                      resource services should call this and pass
+ *                      resource services should call this and pass along supported params
  * @returns Search set result bundle
  */
-const baseSearch = async (args, { req }, resourceType, paramDefs = {}) => {
+const baseSearch = async (args, { req }, resourceType, paramDefs) => {
+  logger.info(`${resourceType} >>> search`);
   // grab the schemas for the data type and Bundle to use for response
   const dataType = resolveSchema(args.base_version, resourceType.toLowerCase());
   const Bundle = resolveSchema(args.base_version, 'bundle');
 
+  // Reprsents search params retrieved from fhir spec, or custom params specified by resource service
```

Comment:
```suggestion
  // Represents search params retrieved from fhir spec, or custom params specified by resource service
```

Reply:
This typo was intentional, suggestion declined.